.github/workflows/actions/use-substrate/README.md

```diff
@@ -0,0 +1,3 @@
+# use-substrate
+
+This action downloads the substrate binary produced from the `build-substrate` workflow and puts it into the `$PATH` so that it can be used by the following steps.
```

Comment:
I don't the `following steps` part of the README and I think the last pattern after `$PATH` can removed anyway

Reply:
Yeah that's fair :)